source/libtomo/misc/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+
+# Locate sources for this project headers are included for IDEs
+file(GLOB misc_tomo_sources
+    ${CMAKE_CURRENT_LIST_DIR}/*.c)
+
+file(GLOB misc_tomo_headers
+    ${CMAKE_CURRENT_LIST_DIR}/../../include/misc/*.h)
+
+file(GLOB misc_tomo_headers
+    ${CMAKE_CURRENT_LIST_DIR}/../../include/misc/*.h)
+
+# core sources
+set(MISC_LIBRARY_SOURCES ${misc_tomo_headers} ${misc_tomo_sources})
+
+# create library
+add_library(libtomopy-misc SHARED ${MISC_LIBRARY_SOURCES})
+
+# link library
+target_link_libraries(libtomopy-misc PUBLIC ${EXTERNAL_LIBRARIES}
+    PRIVATE ${EXTERNAL_PRIVATE_LIBRARIES})
```

Comment:
Libraries like misc do not have any external dependencies, so they should not be linked against `${EXTERNAL_LIBRARIES}`.

Reply:
When I remove the call to `${EXTERNAL_LIBRARIES}` and build, I get an error saying I have undefined functions liked `downsample`, `upsample`, `atan2`, and `sincos`.